apps/SageAccountWeb/src/components/OAuthClientManagementPage.tsx

```diff
@@ -19,9 +19,9 @@ export function OAuthClientManagementPageInternal() {
           },
         }}
       >
-        <BackButton to={'/authenticated/myaccount'} />
-        <Box sx={{ padding: '60px' }}>
-          <SourceAppLogo sx={{ display: 'block' }} />
+        <Box sx={{ padding: '60px', position: 'relative' }}>
```

Comment:
https://sagebionetworks.jira.com/browse/PORTALS-3142

Reply:
Found all occurrences of the BackButton, and verified that they were rendered in the expected position.